src/components/maps/SelectOfficeDropdown.tsx

```diff
@@ -61,7 +74,7 @@ export default function SelectOfficeDropdown({
           <MenuItem
             key={index}
             value={option.title}
-            onClick={() => handleMenuItemClick(option.title)}
+            onClick={() => handleMenuItemClick(option.title||"")}
```

Comment:
since `useMapData` handles empty strings we're ok always calling the handler, but it would be wiser to use `option.title ?? undefined`  if for some reason this becomes nullable